### ISSUE TITLE
WPTs need to be updated to reflect that various metrics have been removed (moved to provisional)

### ISSUE THREAD

**Issue by henbos** [w3c/webrtc-stats#647]
Probably easiest to remove tests for unimplemented provisional metrics, in case they get implemented in the future we should add tests as part of those efforts

**Comment by dontcallmedom** [comment]
I've reviewed the tests in  https://github.com/web-platform-tests/wpt/tree/70bf8f1717e4198f00a8b0aaa19af440c6f48869/webrtc-stats - 2 of them are automatically kept in sync with the IDL in the spec, and the 3 others are not impacted by the recent changes.

I've filed https://github.com/web-platform-tests/wpt/pull/35703 to update the MTI tests associated to the webrtc spec, for which a review would be welcomed.

**Comment by vr000m** [comment]
@dontcallmedom should we close this issue? given the other work was merged?

**Comment by dontcallmedom** [comment]
yes from my perspective, but probably worth checking if @henbos had other changes in mind to track with this issue
